znai-reactjs/src/utils/json.js

```diff
@@ -18,11 +18,14 @@ import * as Promise from 'promise'
 
 import {fullResourcePath} from './resourcePath'
 
-export {jsonPromise}
+export {jsonPromise, jsonPromiseWithUrl}
 
 function jsonPromise(docMeta, url) {
     url = fullResourcePath(docMeta.id, url)
+    return jsonPromise(url)
+}
 
+function jsonPromiseWithUrl(url) {
```

Comment:
Not fully follow here, could you please explain?

Reply:
I want to export two jsonPromise()'s but I can't export the same method name with different signature. So I refactored code. Now jsonPromise gets only one parameter(url).